architecture/topics/aws_getting_started_before.adoc

```diff
@@ -0,0 +1,210 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-before-{context}']
+= Before you begin deploying the AWS Service Broker
+
+Before you can get started deploying the AWS Service Broker, make sure you meet the prerequisites described in this topic.
+
+.Prerequisites 
+
+The following items are required to deploy the AWS Service Broker into {product-title}.
+
+{product-title}::
+You must have an {product-title} cluster configured to run the Service Catalog.
+
+You must configure an {product-title} persistent volume (PV) for use by AWS Service Broker (1 GiB recommended).
```

Comment:
I would check with @jaymconn if this requirement for a 1GB PV is still required by the broker.  
(My guess is this is an old requirement from when automation-broker was used and it needed it's own etcd)